src/Service.Tests/SqlTests/GraphQLMutationTests/DwSqlGraphQLMutationTests.cs

```diff
@@ -0,0 +1,604 @@
+            string graphQLMutation = @"
+                mutation {
+                    createbook(item: { id: 1, title: ""My New Book"", publisher_id: 1234 }) {
+                        result
+                    }
+                }
+            ";
+
+            JsonElement actual = await ExecuteGraphQLRequestAsync(graphQLMutation, graphQLMutationName, isAuthenticated: true);
+            Assert.IsTrue(actual.ToString().Contains("success"), "DwSql mutation insert should be successful");
+        }
+
+        /// <summary>
+        /// <code>Do: </code> Inserts new book using variables to set its title and publisher_id
+        /// <code>Check: </code> If book with the expected values of the new book is present in the database and
+        /// if the mutation query has returned the correct information
+        /// </summary>
+        [TestMethod]
+        public async Task InsertMutationWithVariables()
+        {
+            string graphQLMutationName = "createbook";
+            string graphQLMutation = @"
+                mutation($title: String!, $publisher_id: Int!) {
+                    createbook(item: { id: 2, title: $title, publisher_id: $publisher_id }) {
+                        result
+                    }
+                }
+            ";
+
+            JsonElement actual = await ExecuteGraphQLRequestAsync(graphQLMutation, graphQLMutationName, isAuthenticated: true, new() { { "title", "My New Book" }, { "publisher_id", 1234 } });
+            Assert.IsTrue(actual.ToString().Contains("success"), "DwSql mutation insert with variables should be successful");
+        }
+
+        /// <summary>
+        /// <code>Do: </code>Inserts a new book in database and returns the typename
+        /// <code>Check: </code>if the mutation executed successfully and returned the correct typename
+        /// </summary>
+        [TestMethod]
+        public override async Task InsertMutationWithOnlyTypenameInSelectionSet()
+        {
+            string graphQLMutationName = "createbook";
+            string graphQLMutation = @"
+                mutation {
+                    createbook(item: { id: 1, title: ""Awesome Book"", publisher_id: 1234 }) {
+                        __typename
+                    }
+                }
+            ";
+
+            JsonElement actual = await ExecuteGraphQLRequestAsync(graphQLMutation, graphQLMutationName, isAuthenticated: true);
+            string expected = @"
+              {
+                ""__typename"": ""DbOperationResult""
+              }
+            ";
+
+            SqlTestHelper.PerformTestEqualJsonStrings(expected, actual.ToString());
+        }
+
+        /// <summary>
+        /// <code>Do: </code>Update book in database and return its updated fields
+        /// <code>Check: </code>if the book with the id of the edited book and the new values exists in the database
+        /// and if the mutation query has returned the values correctly
```

Comment:
all the test comments in this file should be updated to reflect that "return its updated fields" will not occur for DWSQL mutations. 

The "check" section should only really mention that a result value of "success" is verified. These tests aren't actually verifying the state of the db.

Reply:
fixed.